runtime/perfetto/db.go

```diff
@@ -120,14 +130,16 @@ func Open(ctx context.Context, fname string) (*DB, error) {
 	}
 
 	t := &DB{
+		opts:            opts,
 		fname:           fname,
 		db:              db,
 		replicaNumCache: cache,
 	}
 
-	const initTables = `
+	initTables := `
 -- Trace data.
 CREATE TABLE IF NOT EXISTS traces (
+	insertion_time INTEGER NOT NULL,
```

Comment:
Is there some ordering imposed here that would allow the DB to find and delete old records in time proportional to the number of old records (and not proportional to the DB size)?